app/src/main/java/org/sopt/sample/ui/main/home/HomeFragment.kt

```diff
@@ -0,0 +1,56 @@
+package org.sopt.sample.ui.main.home
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import org.sopt.sample.R
+import org.sopt.sample.databinding.FragmentHomeBinding
+
+class HomeFragment : Fragment() {
+    private var _binding: FragmentHomeBinding? = null
+    private val binding get() = requireNotNull(_binding) { "${this::class.java.simpleName}에서 바인딩 초기화 에러가 발생했습니다." }
+    private lateinit var adapter: InstagramAdapter
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentHomeBinding.inflate(layoutInflater)
+        initAdapter()
+        setList()
+        return binding.root
+    }
+
+    private fun initAdapter() {
+        adapter = InstagramAdapter()
+        binding.recyclerViewHome.adapter = adapter
+    }
+
+    private fun setList() {
+        val list: List<InstagramData> = listOf(
+            InstagramTitle(title = getString(R.string.instagram_followers)),
+            InstagramContent(name = "권용민", id = "@k_dragonm"),
+            InstagramContent(name = "김수빈", id = "@subeenie0608"),
+            InstagramContent(name = "최윤정", id = "@c_yun_east"),
+            InstagramContent(name = "최유리", id = "@uxri___"),
+            InstagramContent(name = "이준원", id = "@junwon1108"),
+            InstagramContent(name = "김준우", id = "@???????"),
+            InstagramContent(name = "김지영", id = "@zzi.__.0"),
+            InstagramContent(name = "김우남", id = "unam_0107"),
+            InstagramContent(name = "김세훈", id = "@s2ehun"),
+            InstagramContent(name = "이현우", id = "@I2hyunwoo"),
```

Comment:
```suggestion
            InstagramContent(name = "이현우", id = "@l2hyunwoo"),
```

I 아니고 l입니다 (아이 아니고 엘입니다)